wolfhsm/wh_cryptocb.h

```diff
@@ -0,0 +1,35 @@
+/* cryptocb.h
+ *
+ * Copyright (C) 2006-2023 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
```

Comment:
All of wolfHSM is actually under GPLv3, not 2.  We'll go back and fix all of the licensing once we are mostly feature complete.